pkg/schedule/filter/region_filters.go

```diff
@@ -172,6 +172,16 @@ func (f *regionWitnessFilter) Select(region *core.RegionInfo) *plan.Status {
 	return statusOK
 }
 
+// HasWitnessPeers checks if the region has witness peers.
+func HasWitnessPeers(region *core.RegionInfo) bool {
```

Comment:
Where is this used?

Reply:
Do we need to consider the witness?

Reply:
Concerned about the impact of the witness, the region containing the witness peer will be skipped during the affinity check.

Reply:
so we can remove all changeds related with witness